rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -238,13 +239,34 @@ public Response startLRA(
                     return Response.status(response.getStatus()).build();
                 }
             } else {
-                ResponseHolder resp = new RequestBuilder(parentLRAUrl)
+                LRALogger.i18NLogger.error_parentLRANotFound(parentLRAUrl);
```

Comment:
Failure to find a resource in a distributed system is not an error (in fact it is a common occurrence).

Reply:
But I do notice that if the LRA cannot be registered with the parent then we should be closing the one that was just started on the previous line. I will raise an issue to get that fixed.